src/pages/MonthlyGood/MonthlyGoodSetupPage.vue

```diff
@@ -392,6 +399,7 @@ export default {
 				});
 		},
 		result({ data }) {
+			this.isMGTaglineActive = _get(data, 'general.mg_tagline_active.value') === 'true' || false;
```

Comment:
this should start with `ui` not `general`
we have special handling for graphql and the `uiConfigSetting` that expects a setting to be named like `ui.mg_tagline_active` 

We should also add the word "donation" somewhere in your setting key so we have some idea where the setting applies in the mg page...